cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/hvdc/DcLineSegmentConversion.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) 2017-2018, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.cgmes.conversion.elements.hvdc;
+
+import java.util.Objects;
+
+import com.powsybl.cgmes.conversion.Context;
+import com.powsybl.cgmes.conversion.elements.AbstractIdentifiedObjectConversion;
+import com.powsybl.iidm.network.HvdcLine;
+import com.powsybl.iidm.network.HvdcLineAdder;
+import com.powsybl.triplestore.api.PropertyBag;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public class DcLineSegmentConversion extends AbstractIdentifiedObjectConversion {
+
+    private static final double DEFAULT_MAXP_FACTOR = 1.2;
+
+    DcLineSegmentConversion(PropertyBag l, HvdcLine.ConvertersMode mode, double r, double ratedUdc,
+        DcLineSegmentConverter converter1, DcLineSegmentConverter converter2, Context context) {
+        super("DCLineSegment", l, context);
+
+        Objects.requireNonNull(converter1);
+        Objects.requireNonNull(converter2);
+        this.mode = mode;
+        this.r = r;
+        this.ratedUdc = ratedUdc;
+        this.converter1 = converter1;
+        this.converter2 = converter2;
+    }
+
+    @Override
+    public boolean valid() {
+        return true;
+    }
+
+    @Override
+    public void convert() {
+
+        // arbitrary value because there is no maxP attribute in CGMES
+        double maxP = getMaxP(converter1.pAC, converter2.pAC, mode);
+        missing("maxP", maxP);
+
+        HvdcLineAdder adder = context.network().newHvdcLine()
+            .setR(r)
+            .setNominalV(ratedUdc)
+            .setActivePowerSetpoint(
+                getPDc(converter1.pAC, converter2.pAC, converter1.poleLossP, converter2.poleLossP, mode))
+            .setMaxP(maxP)
+            .setConvertersMode(mode)
+            .setConverterStationId1(converter1.converterId)
+            .setConverterStationId2(converter2.converterId);
+        identify(adder);
+        adder.add();
+    }
+
+    private static double getMaxP(double pAC1, double pAC2, HvdcLine.ConvertersMode mode) {
```

Comment:
@geofjamg and @mathbagu I think that ````MaxP```` should be always positive too as the active power set point ? It is not the case here. so we maybe have to fix the computation.

Reply:
@geofjamg and @mathbagu I still have a question about ````PowerFactor```` and `````LossFactor```` that are documented as in percent unit (https://www.powsybl.org/docs/iidm/model/lccConverterStation.html). But it seems to be ratios in other parts of powsybl-core. Is it a ratio or a percentage ?